grace/models/classifier.py

```diff
@@ -26,14 +26,16 @@ def __init__(
         input_channels: int,
         hidden_channels: int,
         *,
-        output_classes: int = 2,
+        node_output_classes: int = 2,
+        edge_output_classes: int = 2,
     ):
         super(GCN, self).__init__()
         torch.manual_seed(12345)
         self.conv1 = GCNConv(input_channels, hidden_channels)
         self.conv2 = GCNConv(hidden_channels, hidden_channels)
         self.conv3 = GCNConv(hidden_channels, hidden_channels)
-        self.linear = Linear(hidden_channels, output_classes)
+        self.node_classifier = Linear(hidden_channels, node_output_classes)
+        self.edge_classifier = Linear(hidden_channels * 2, edge_output_classes)
```

Comment:
This is because you cat the source and target node embeddings?

Reply:
Maybe add a comment here to describe the logic. Better still, add a Notes section to the docstrings